src/plugins/workspace/public/components/workspace_form/use_workspace_form.ts

```diff
@@ -149,6 +148,9 @@ export const useWorkspaceForm = ({
       },
       false
     );
+    if (result?.success) {
+      setPermissionSettings(settings);
+    }
   };
```

Comment:
nit: could simplified to check result only, as `success` is always true based on its definition

`Promise<{ result: boolean; success: true } | undefined>;`

Reply:
Updated. Thanks.